File: back-end/src/main/java/com/spring/resturant/model/Order.java
package com.spring.resturant.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name ="orderfood")
public class Order extends CategoryOrder {

    @Column(name = "price")
    private int price;

    @Column(name = "image")
    private String img;

    @Lob
    @Column(name = "description")
    private String description;

    @ManyToOne
    @JoinColumn(name = "category_id")
    private Category category;
}

File: front-end/src/app/model/order.ts
export class Order {

  id: number;
  name: string;
  price: number;
  img: string;
  description: string;
  data_create: Date;
  data_update: Date;

  constructor(id: number, name: string, price: number, img: string, description: string, data_create: Date, data_update: Date) {
    this.id = id;
    this.name = name;
    this.price = price;
    this.img = img;
    this.description = description;
    this.data_create = data_create;
    this.data_update = data_update;
  }
}
